content/agent/kubernetes/helm.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Deploying Datadog in Kubernetes using Helm 
+kind: documentation
+further_reading:
+- link: "agent/kubernetes/daemonset_setup"
+  tag: "documentation"
+  text: "Kubernetes DaemonSet Setup"
+- link: "agent/kubernetes/host_setup"
+  tag: "documentation"
+  text: "Kubernetes Host Setup"
+- link: "agent/kubernetes/metrics"
+  tag: "documentation"
+  text: "Kubernetes Metrics"
+---
+
+Helm is a package management tool for Kubernetes. 
+
+## Installing Helm
+
+### Installing the Helm client
+
+{{< tabs >}}
+{{% tab "macOS (Homebrew)" %}}
+
+```bash
+brew install kubernetes-helm
+```
+
+{{% /tab %}}
+{{% tab "Linux (Snap)" %}}
+
+```bash
+sudo snap install helm --classic
+```
+
+{{% /tab %}}
+{{% tab "Windows (Chocolatey)" %}}
+
+```bash
+choco install kubernetes-helm
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+For other platforms and methods of installing Helm, refer to the [Helm documentation][1].
+
+### Installing the Helm server (Tiller)
+
+If your Kubernetes environment does not use RBAC, the following command installs Tiller in your cluster:
+
+```bash
+helm init
+```
+
+Refer to [Helm's Tiller documentation][2] for further details.
+
+If your Kubernetes cluster is RBAC-enabled, use the following RBAC to deploy Tiller.
+
+```yaml
+apiVersion: v1  
+kind: ServiceAccount  
+metadata:  
+ name: tiller  
+ namespace: kube-system  
+---  
+apiVersion: rbac.authorization.k8s.io/v1beta1  
+kind: ClusterRoleBinding  
+metadata:  
+ name: tiller  
+roleRef:  
+ apiGroup: rbac.authorization.k8s.io  
+ kind: ClusterRole  
+ name: cluster-admin  
+subjects:  
+ - kind: ServiceAccount  
+   name: tiller  
+   namespace: kube-system
+```
+
+With this as your `tiller-rbac-config.yaml`, then run:
+
+```bash
+kubectl create -f tiller-rbac-config.yaml
+
+helm init --service-account tiller
+```
+
+Refer to [Helm's Tiller/RBAC documentation][3] for further details.
+
+### Verify installation
+
+To verify your installation, run:
+
+```bash
+kubectl get pods -n kube-system
+```
+
+## Installing the Datadog Helm chart
+
+To install the chart with the release name `<RELEASE_NAME>`, retrieve your Datadog API key from your [Agent installation instructions][4] and run:
+
+```bash
+helm install --name <RELEASE_NAME> --set datadog.apiKey=<DATADOG_API_KEY> stable/datadog
+```
+
+Refer to the section below for a list of configurable parameters.
+
+This chart adds the Datadog Agent to all nodes in your cluster via a DaemonSet. It also optionally depends on the [kube-state-metrics chart][5]. A few minutes after installation, Datadog begins to report hosts and metrics.
+
+### Configuration
+
+The following table lists the configurable parameters of the Datadog chart and their default values.
```

Comment:
I'm not sure we should list these here. Maybe just point to our chart's [readme](https://github.com/helm/charts/tree/master/stable/datadog)? This list gets updated frequently, it will fall out of sync quickly, I guarantee it.